packages/design/src/components/message-box/_variables.scss

```
@@ -0,0 +1,14 @@
+$messagebox-info-color-background: var(--fkds-color-feedback-background-info);
+$messagebox-info-color-border: var(--fkds-color-feedback-border-info);
+
+$messagebox-error-color-background: var(--fkds-color-feedback-background-negative);
+$messagebox-error-color-border: var(--fkds-color-feedback-border-negative);
+
+$messagebox-warning-color-background: var(--fkds-color-feedback-background-warning);
+$messagebox-warning-color-border: var(--fkds-color-feedback-border-warning);
+
+$messagebox-success-color-background: var(--fkds-color-feedback-background-positive);
+$messagebox-success-color-border: var(--fkds-color-feedback-border-positive);
+
+$messagebox-info-color-heading: var(--fkds-color-text-primary);
+$messageBox-error-icon-error-color-border: var(--fkds-icon-color-feedback-content-negative);
```

```suggestion
$messagebox-error-icon-error-color-border: var(--fkds-icon-color-feedback-content-negative);
```

och motsvarande ändring i `_message-box.scss`